File: Threetier without factorypattern/ConApp/Authentication.cs
using BusinessLayer;
using DomainLayer.Models;

namespace ConApp
{
    class Authentication
    {
        AuthencationBusiness _authBusiness;
        public Authentication()
        {
            _authBusiness = new AuthencationBusiness();
        }
        internal bool Login(LoginModel loginModel)
        {
            return _authBusiness.ValidateLogin(loginModel);
        }
        internal bool Register(RegistrationModel registrationModel)
        {
            return _authBusiness.ValidateRegister(registrationModel);
        }
    }
}

File: Inventory/src/Inventory/Authentication.java
package Inventory;
import Domain.*;
import Bussiness.*;

public class Authentication {
	AuthencationBusiness _authBusiness;
    public Authentication()
    {
        _authBusiness = new AuthencationBusiness();
    }
    public boolean Login(LoginModel loginModel)
    {
        return _authBusiness.ValidateLogin(loginModel);
    }
    public boolean Register(RegistrationModel registrationModel)
    {
        return _authBusiness.ValidateRegister(registrationModel);
    }
}
